### DISCUSSION TITLE
Drop everything via an API endpoint

### DISCUSSION THREAD

**Discussion by TheRyan1** [Mimetis/Dotmim.Sync#995] [Q&A] [answered]
Hello, I would like to know if it's possible to drop all the scope info, stored procedures and tracking tables on the server via an endpoint. For example. If my server is deployed and syncing has happened between server and client already, is it possible to hit an endpoint that calls the DropAllAsync method on the server DB? 

Thanks in advance

**Comment by Mimetis** [comment, marked as answer]
I moved this to conversation as it's not an issue

BTW, you can create your own endpoint and call the `DropAllAsync()` method if you want, independently from **DMS**:

You can use this kind of code I guess

``` cs
public SyncController(WebServerAgent webServerAgent, ILogger<SyncController> logger)
{
    this.webServerAgent = webServerAgent;
    this.logger = logger;
}

[HttpGet("drop")]
public async Task<IActionResult> DropAllAsync()
{
    await this.webServerAgent.RemoteOrchestrator.DropAllAsync();
    return Ok();

}

```

**Reply by TheRyan1** [reply]
Thank you so much for answering and moving it over to the discussions. This is actually what I originally have done but was not sure whether or not it would have some knock on effects later on. Thanks again! Great work with the library by the way. It's been wonderful using it! Have a great day.

**Reply by Mimetis** [reply]
That's being said, be careful with this endpoint, be sure it's protected to ensure no users not allowed will be able to call it !
